modules/administration/pages/custom-ssl.adoc

```diff
@@ -1,97 +1,289 @@
+In the second field, after the ``@``, enter [guimenu]``*``.
+Click btn:[Find Targets].
+. Check that the appropriate clients are displayed, and click btn:[Run Command] to deploy the certificates on the clients.
 
+image::deploy_cert_salt.png[scaledwidth=80%]
 
-After completing the installation with yast found in <<advanced.topics.proxy.quickstart>> continue with a modified <<at.manager.proxy.run.confproxy>> procedure:
+When you have every client trusting both the old and new certificates, you can go ahead and replace the server certificate on the {productname} Server and Proxies.
 
-[[bp.cert.custom.proxy.proc]]
 
-. Execute [command]``configure-proxy.sh``.
-. When prompted with:
+
+.Procedure: Replace Server Certificate on the Server
+
+. On the {productname} Server, at the command prompt, install the RPM from the [path]``ssl-build`` directory:
++
+----
+rpm -Uhv ssl-build/susemanager/rhn-org-httpd-ssl-key-pair-susemanager-1.0-2.noarch.rpm
+----
+. Restart services to pick the changes:
 +
+----
+spacewalk-service restart
+----
+
+
+
+.Procedure: Replace Server Certificate on the Proxy
 
+. On the {productname} Proxy, at the command prompt, install the RPM from the [path]``ssl-build`` directory:
++
 ----
-Do you want to import existing certificates?
+rpm -Uhv ssl-build/susemanager-proxy/rhn-org-httpd-ssl-key-pair-susemanager-proxy-1.0-2.noarch.rpm
 ----
+. Restart services to pick up the changes:
 +
-Answer with kbd:[y]
-.
-. Continue by following the script prompts.
+----
+spacewalk-service restart
+----
+
+
+When you have replaced the server certificates on your server and any proxies, you need to update the certificate with only the new details all the clients.
+This is done by adding it to the client channels you set up previously.
+
+
+
+.Procedure: Adding the New Certificates to the Client Channel
+
+. Copy the combined certificate RPM into the [path]``/root/ssl-build/`` directory:
++
+----
+cp /root/combined-ssl-build/*.rpm /root/ssl-build/
+----
+. Generate a new RPM with from the new certificates.
+Check the release number carefully to ensure you have the right certificate file:
++
+----
+rhn-ssl-tool --gen-ca --rpm-only --dir="/root/ssl-build"
+----
+. Install the new local certificates on the {productname} Server:
++
+----
+/usr/bin/rhn-deploy-ca-cert.pl --source-dir /root/ssl-build \
+--target-dir /srv/www/htdocs/pub/ --trust-dir=/etc/pki/trust/anchors/
+----
+. Restart services to pick up the changes:
++
+----
+spacewalk-service restart
+----
+. Upload the new RPM into the channel:
++
+----
+rhnpush -c ssl-ca-channel --nosig \
+--ca-chain=/srv/www/htdocs/pub/RHN-ORG-TRUSTED-SSL-CERT \
+/root/ssl-build/rhn-org-trusted-ssl-cert-1.0-3.noarch.rpm
+----
+
+
+When you have the new certificate in the channel, you can use the {productname} {webui} to update it on all clients by synchronizing them with the channel.
+Alternatively, for Salt clients, you can use menu:Salt[Remote Commands].
```

Comment:
Maybe you can reference the "Deploying Combined Certificates on Salt Clients" section.